internal/file/digest.go

```diff
@@ -0,0 +1,35 @@
+package file
+
+import (
+	"crypto/sha1" //nolint:gosec
+	"encoding/hex"
+	"io"
+	"os"
+
+	"golang.org/x/xerrors"
+
+	"github.com/anchore/syft/internal/log"
+)
+
+const DefaultDigestAlgorithm = "sha1"
```

Comment:
I think I'm not quite following why this const exists. I was looking at the usages through the code, and I'm wondering why we would refer to this value as opposed to the `Digest.Algorithm` that's attached to the digest value. Wouldn't that be the source of truth for which algorithm was used for the digest?

Reply:
This is definitely a holdover from the fact that this was the first file written vs what came out at the end. Looking again at the whole PR it probably makes sense to not play favorites here on the algorithms.

We currently have `Digest` singular on the javametadata. Looking at how this gets translated into some other specs Digests becomes plural in most cases. 

I'm going to update `Javametadata` so that `Digest` --> `Digests` so we can eliminate the incongruency and not have this default.